DOC-2135: Note - not possible to reconfigure existing non-validating notary into validating one (#444)
corda/corda-docs-portal: content/en/platform/corda/4.8/enterprise/operations/deployment/corda-configuration-fields.md

For a single-node notary, you must specify the `validating` and `serviceLegalName` configuration fields.
{{< /warning >}}

{{< note >}} 
Once a notary is configured with a default value, it cannot be reconfigured. To change a non-validating notary to validating

> (or vice-versa) requires a new notary creation and registration. 

Reviewer: ```suggestion
(or vice-versa), you need to create and register a new notary. 
```
